packages/main/src/List.js

```diff
@@ -93,7 +93,7 @@ const metadata = {
 		},
 
 		/**
-		 * Determines whether the list items are indented.
+		 * Determines whether the <code>ui5-list</code> is indented.
```

Comment:
At some point we stared using "component" when referring to the current web component, because if the List is inherited by other web component (ui5-my-list), the docs will mention ui5-list, which won't be true. So "component" is more safe